server.js

```diff
@@ -15,6 +15,15 @@ app.use(cors());
 
 app.set('port', process.env.PORT || 3001);
 
+app.get('/api/v1/races', async (request, response) => {
+  try {
+    const races = await database('races').select();
+    response.status(200).json(races);
```

Comment:
2. Get races from `races` table in DB and if it's successful all races array is returned in response with status code `200`